src/yunohost/backup.py

```diff
@@ -602,10 +616,21 @@ def backup_info(name, with_details=False, human_readable=False):
 
     """
     archive_file = '%s/%s.tar.gz' % (archives_path, name)
-    if not os.path.isfile(archive_file):
+
+    # Check file exist (even if it's a broken symlink)
+    if not os.path.lexists(archive_file):
```

Comment:
It seems to be `os.path.exists` according [to this](https://stackoverflow.com/questions/15077424/python-os-path-exists-vs-os-path-isdir).

So, a typo, an `l` surplus.